README.md

```diff
@@ -432,6 +433,27 @@ example `<Heading.h1>`, `<Heading.h2>`, `<Heading.h3>` etc.
 
 [View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Heading/story.js)
 
+### HighlightedName
+
+By default the background color of a `HighlightedName` component is generated
+automatically from its `children` (which must be a string), though this can be overridden.
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/HighlightedName/story.js)
+
+#### Props
+
+| Name          | Type      | Default | Required | Description                                |
+| ------------- | --------- | ------- | -------- | ------------------------------------------ |
+| `primary`     | `boolean` | -       | -        | If true, use the `primary` theme color     |
+| `secondary`   | `boolean` | -       | -        | If true, use the `secondary` theme color   |
+| `tertiary`    | `boolean` | -       | -        | If true, use the `tertiary` theme color    |
+| `quaternary`  | `boolean` | -       | -        | If true, use the `quaternary` theme color  |
```

Comment:
I would actually remove all props except for `children`. It is better to start with a minimum API, and add on top of it if truly necessary (since we already support `bg` and `color`).